packages/workbox-background-sync/lib/StorableRequest.mjs

```diff
@@ -0,0 +1,119 @@
+/*
+ Copyright 2017 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import '../_version.mjs';
+
+const serializableProperties = [
+  'method',
+  'referrer',
+  'referrerPolicy',
+  'mode',
+  'credentials',
+  'cache',
+  'redirect',
+  'integrity',
+  'keepalive',
+  'signal',
+];
+
+
+/**
+ * A class to make it easier to serialize and de-serialize requests so they
+ * can be stored in IndexedDB.
+ */
+export default class StorableRequest {
+  /**
+   * Converts a Request object to a plain object that can be structured
+   * cloned or JSON-stringified.
+   *
+   * @param {Request} request
+   * @return {Promise<StorableRequest>}
+   */
+  static async fromRequest(request) {
+    const requestInit = {headers: {}};
+
+    // Set the body if present.
+    if (request.method != 'GET') {
```

Comment:
Could you switch here and elsewhere to `!==`? (Shouldn't our linting pick that up?)

Reply:
I believe the linting only warns against double-equals comparison with falsy values (as that's the only time it gets tricky).

I can change here, but I think we should add this to our style guide if we're going to enforce it, as it's not part of Google JS style.